tests/foreman/cli/test_contentviews.py

```diff
@@ -315,6 +318,180 @@ def test_delete_cv_version(self):
         result = ContentView.version_delete({u'id': version1_id})
         self.assertEqual(result.return_code, 0)
 
+    def test_remove_cv_environment(self):
+        """@Test: Remove content view from lifecycle environment assignment
+
+        @Feature: Content Views
+
+        @Assert: Content view removed from environment successfully
+
+        """
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': new_cv['id']})
+        result = ContentView.info({u'id': new_cv['id']})
+        env = result.stdout['lifecycle-environments'][0]
+
+        result = ContentView.remove({
+            u'id': new_cv['id'],
+            u'environment-ids': env['id'],
+            u'organization-id': new_org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentView.info({u'id': new_cv['id']})
+        self.assertFalse(result.stdout['lifecycle-environments'])
```

Comment:
better to check the length: `self.assertEqual(len(result.stdout['lifecycle-environments']), 0). This will give more meaning when someone else is reading the test.